src/analytics/formResponseTracker/formResponseTracker.test.ts

```diff
@@ -0,0 +1,204 @@
+import { describe, expect, jest, test } from "@jest/globals";
+import { FormResponseTracker } from "./formResponseTracker";
+import { FormEventInterface } from "../formTracker/formTracker.interface";
+window.DI = { analyticsGa4: { cookie: { consent: true } } };
+
```

Comment:
we have tests for text input, dropdown ,textarea and checkbox. Do we need one for radio buttons?

Reply:
We have already a test for dropdown. line 150.

Reply:
yes but for radio buttons ?

Reply:
Oups sorry! Well spotted! I added it!

Reply:
fixed